Casks/b/bepo.rb

```diff
@@ -2,15 +2,14 @@
   version "1.1rc2"
   sha256 "4c0cb9e058a5bdc3de8b6056525046c72976afad9b88298b74a9854cb427d56c"
 
-  url "https://download.tuxfamily.org/dvorak/macosx/fr-dvorak-bepo-macosx-#{version}.dmg",
-      verified: "download.tuxfamily.org/dvorak/macosx/"
+  url "https://cdn.bepo.fr/macosx/fr-dvorak-bepo-macosx-#{version}.dmg"
   name "Bépo layout"
   desc "Keyboard layout designed to facilitate input of French and computer languages"
   homepage "https://bepo.fr/"
 
   livecheck do
-    url "https://download.tuxfamily.org/dvorak/macosx/"
-    regex(/href=.*?fr[._-]dvorak[._-]bepo[._-]macosx[._-]v?(\d+(?:\.\d+)+(?:rc\d+)?)\.dmg/i)
+    url "https://bepo.fr/wiki/Mod%C3%A8le:Version_actuelle/Mac"
+    regex(%r{href="/wiki/Version_?(\d+(?:\.\d+)+(?:rc\d+)?)"}i)
```

Comment:
```suggestion
    regex(/href=.*?fr[._-]dvorak[._-]bepo[._-]macosx[._-]v?(\d+(?:\.\d+)+(?:rc\d+)?)\.dmg/i)
```

We can keep the `regex` unchanged, since the wiki page also links to the binary.

Reply:
I was afraid the link would conflict with something else if the template gets changed, but sure